asteroid/filterbanks/multiphase_gammatone_filterbank.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+import torch
+from .enc_dec import Filterbank
+
+class MultiphaseGammatoneFB(Filterbank):
+    """ Multi-Phase Gammatone Filterbank as described in [1].
+        Please cite [1] whenever using this.
+        Original code repository: `<https://github.com/sp-uhh/mp-gtf>`
+
+    References:
+    [1] David Ditter, Timo Gerkmann, "A Multi-Phase Gammatone Filterbank for Speech
+        Separation via TasNet", IEEE Int. Conf. Acoust., Speech, Signal Process. (ICASSP),
+        Barcelona, Spain, May 2020.
+        Available: `<https://ieeexplore.ieee.org/document/9053602/>`
+
+    Args:
+        n_filters (int): Number of filters. 
+        kernel_size (int): Length of the filters.
+        sample_rate (int, optional): The sample rate (used for initialization).
+        stride (int, optional): Stride of the convolution. If None (default),
+            set to ``kernel_size // 2``.
+    """
+    
+    def __init__(self, n_filters=128, kernel_size=16, sample_rate=8000, stride=None):
+        super(MultiphaseGammatoneFB, self).__init__(n_filters, kernel_size, stride=stride)
+        self.sample_rate = sample_rate
+        self.n_filters = n_filters
+        self.kernel_size = kernel_size
```

Comment:
`n_filters`, `kernel_size` and `stride` are registered in the class in  `Filterbank`, no need to add these lines.

Reply:
Ok, deleted the 2 lines.